<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::ManageQuickPicksListDialog</class>
 <widget class="QDialog" name="tt3::gui::ManageQuickPicksListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage quick picks list</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Misc/QuickPicksLarge.png</normaloff>:/tt3-gui/Resources/Images/Misc/QuickPicksLarge.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tasksTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="publicActivitiesTab">
      <attribute name="icon">
       <iconset resource="tt3-gui.qrc">
        <normaloff>:/tt3-gui/Resources/Images/Objects/PublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Objects/PublicActivitySmall.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Public activities</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="publicActivitiesFilterWidget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="publicActivitiesFilterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="publicActivitiesFilterLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="publicActivitiesTreeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="publicTasksTab">
      <attribute name="icon">
       <iconset resource="tt3-gui.qrc">
        <normaloff>:/tt3-gui/Resources/Images/Objects/PublicTaskSmall.png</normaloff>:/tt3-gui/Resources/Images/Objects/PublicTaskSmall.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Public tasks</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="publicTasksFilterWidget" native="true">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="publicTasksFilterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="publicTasksFilterLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="publicTasksTreeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="privateActivitiesTab">
      <attribute name="icon">
       <iconset resource="tt3-gui.qrc">
        <normaloff>:/tt3-gui/Resources/Images/Objects/PrivateActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Objects/PrivateActivitySmall.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Private activities</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="privateActivitiesFilterWidget" native="true">
         <layout class="QFormLayout" name="formLayout_3">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="privateActivitiesFilterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="privateActivitiesFilterLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="privateActivitiesTreeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="privateTasksTab">
      <attribute name="icon">
       <iconset resource="tt3-gui.qrc">
        <normaloff>:/tt3-gui/Resources/Images/Objects/PrivateTaskSmall.png</normaloff>:/tt3-gui/Resources/Images/Objects/PrivateTaskSmall.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Private tasks</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="privateTasksFilterWidget" native="true">
         <layout class="QFormLayout" name="formLayout_4">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="privateTasksFilterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="privateTasksFilterLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="privateTasksTreeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="actionsPanel" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="moveDownPushButton">
        <property name="text">
         <string>Move down</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/MoveDownSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/MoveDownSmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="moveUpPushButton">
        <property name="text">
         <string>Move up</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/MoveUpSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/MoveUpSmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="moveToTopPushButton">
        <property name="text">
         <string>Move to top</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/MoveToTopSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/MoveToTopSmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="moveToBottomPushButton">
        <property name="text">
         <string>Move to bottom</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/MoveToBottomSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/MoveToBottomSmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="removePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/RemoveSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/RemoveSmall.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="quickPicksListWidget"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tasksTabWidget</tabstop>
  <tabstop>publicActivitiesFilterLineEdit</tabstop>
  <tabstop>publicActivitiesTreeWidget</tabstop>
  <tabstop>quickPicksListWidget</tabstop>
  <tabstop>moveToTopPushButton</tabstop>
  <tabstop>moveUpPushButton</tabstop>
  <tabstop>moveDownPushButton</tabstop>
  <tabstop>moveToBottomPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>privateActivitiesTreeWidget</tabstop>
  <tabstop>privateTasksTreeWidget</tabstop>
  <tabstop>publicTasksTreeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publicActivitiesTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>_publicActivitiesTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publicTasksTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>_publicTasksTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>privateActivitiesTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>_privateActivitiesTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>privateTasksTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>_privateTasksTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quickPicksListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>tt3::gui::ManageQuickPicksListDialog</receiver>
   <slot>_quickPicksListWidgetCurrentRowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_publicActivitiesTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_publicTasksTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_privateActivitiesTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_privateTasksTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_quickPicksListWidgetCurrentRowChanged(int)</slot>
 </slots>
</ui>
